essentials/gamemode.dsc

```diff
@@ -0,0 +1,81 @@
+gmc_command:
+  type: command
+  name: gmc
+  debug: false
+  description: Adjusts another player's or your gamemode to creative mode.
+  usage: /gmc
+  permission: behrry.essentials.gmc
+  tab completions:
+    1: <server.online_players.parse[name].exclude[<player.name>]>
+  script:
+    - define gamemode creative
+    - inject gamemode_task
+
+gma_command:
+  type: command
+  name: gma
+  debug: false
+  description: Adjusts another player's or your gamemode to adventure mode.
+  usage: /gma
+  permission: behrry.essentials.gma
+  tab completions:
+    1: <server.online_players.parse[name].exclude[<player.name>]>
+  script:
+    - define gamemode adventure
+    - inject gamemode_task
+
+gmsp_command:
+  type: command
+  name: gmsp
+  debug: false
+  description: Adjusts another player's or your gamemode to spectator mode.
+  usage: /gmsp
+  permission: behrry.essentials.gmsp
+  tab completions:
+    1: <server.online_players.parse[name].exclude[<player.name>]>
+  script:
+    - define gamemode spectator
+    - inject gamemode_task
+
+gms_command:
+  type: command
+  name: gms
+  debug: false
+  description: Adjusts another player's or your gamemode to survival mode.
+  usage: /gms
+  permission: behrry.essentials.gms
+  tab completions:
+    1: <server.online_players.parse[name].exclude[<player.name>]>
+  script:
+    - define gamemode survival
+    - inject gamemode_task
+
+gamemode_task:
```

Comment:
tab completions? see `heal_command`

Reply:
This is a task script; the gamemode commands all use tab completions though yeah

Reply:
not entirely sure what I was referencing here